sr_edc_ethercat_drivers/CMakeLists.txt

```diff
@@ -29,7 +29,6 @@ generate_messages(
 
 # catkin_package parameters: http://ros.org/doc/groovy/api/catkin/html/dev_guide/generated_cmake_api.html#catkin-package
 catkin_package(
-        DEPENDS binutils
```

Comment:
Isn't this dependency necessary? Otherwise another package depending on sr_edc_ethercat_drivers that wants to include some of the headers (e.g. to extend a class like [sr_edc](https://github.com/shadow-robot/sr-ros-interface-ethercat/blob/indigo-devel/sr_edc_ethercat_drivers/include/sr_edc_ethercat_drivers/sr_edc.h)) won't be able to find bfd.

Is this causing a cmake warning?